node/tty_test.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+import { assert } from "../testing/asserts.ts";
+import { isatty } from "./tty.ts";
+
+Deno.test("[node/tty isatty] returns true when fd is a tty, false otherwise", () => {
+  assert(Deno.isatty(Deno.stdin.rid) === isatty(Deno.stdin.rid));
```

Comment:
I don't find these test cases to be particularly meaningful, but I'm not sure if we can do better here in std as that would involve spawning pty processes 🤔

Reply:
agree